solc_select/__main__.py

```diff
@@ -84,6 +85,9 @@ def solc_select() -> None:
 
 
 def solc() -> None:
+    if not installed_versions():
+        print("No solc versions found, installing latest")
```

Comment:
I wonder if this extra output may confuse tools that expect/consume a specific solc output

Reply:
We can drop it too; I have no strong preference

Reply:
Removed in [217103e](https://github.com/crytic/solc-select/pull/203/commits/217103e3d6ef3a3db8dd33409a08c7fa7a4fead2)

Reply:
This also prints https://github.com/crytic/solc-select/blob/217103e3d6ef3a3db8dd33409a08c7fa7a4fead2/solc_select/solc_select.py#L200